specification/datalake-store/resource-manager/Microsoft.DataLakeStore/2016-11-01/account.json

```diff
@@ -942,6 +942,113 @@
         },
         "x-ms-odata": "#/definitions/DataLakeStoreAccount"
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DataLakeStore/locations/{location}/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "Account"
+        ],
+        "operationId": "Account_CheckNameAvailability",
```

Comment:
`Account` ->`Accounts`, `Account_CheckNameAvailability` -> `Accounts_CheckNameAvailability`

Reply:
For this one I would like to leave as is for this PR because we currently have existing APIs for "Account_Get", "Account_Create", etc. We will make the fix to "Accounts" in an upcoming API version, 2017-10-01, which has other breaking changes -- as we would like to bundle the breaking changes together.